src/scripts/wildBattle/RouteHelper.ts

```diff
@@ -64,6 +63,26 @@ class RouteHelper {
         return true;
     }
 
+    public static getPossibleRoutePokemon(route: number, region: GameConstants.Region, includeHeadbutt = true): PokemonNameType[] {
+        return RouteHelper.getAvailablePokemonList(route, region, includeHeadbutt);
```

Comment:
Why have we added this method? It's not big enough to should be it's own method

Reply:
It is superfluous, removed